File: nowcoder/BC24_总成绩和平均分计算/BC24.go
package main
 
import (
    "fmt"
)

func main() {
    var a, b, c float32
    fmt.Scanf("%f %f %f", &a, &b, &c)
    sum := a + b + c
    fmt.Printf("%.2f %.2f", sum, sum / 3)
}

File: nowcoder/BC24_总成绩和平均分计算/BC24.c
#include<stdio.h>
int main() {
    float a,b,c,sum;
    scanf("%f %f %f", &a, &b, &c);
    sum = a + b + c;
    printf("%.2f %.2f", sum, sum / 3);
    
    return 0;
}